TableGenerator/output.txt

```diff
@@ -0,0 +1,422 @@
+| Platform | Master<br>(5.0.x&nbsp;Runtime) | 5.0.100 Preview 2<br>(5.0 Runtime) | 5.0.100 Preview 1<br>(5.0 Runtime) | Release/3.1.3XX<br>(3.1.x Runtime) | Release/3.1.2XX<br>(3.1.x Runtime) | Release/3.1.1XX<br>(3.1.x Runtime) | Release/3.0.1xx<br>(3.0.x Runtime) |
```

Comment:
If we check it in, we should add a test to ensure this is up-to-date. Or we would trap people with this file.

Or this could be the build result.

Reply:
I pushed this by mistake. I will delete it if that's OK.